creep.setup.hauler.js

```
@@ -17,12 +17,14 @@ setup.maxCount = function(room){
     let miners = (room.population.typeCount['miner']||0);
     let workers = (room.population.typeCount['worker']||0);
     let cont = room.structures.container.in.length + room.structures.links.storage.length;
-    if( miners > 0  || ( cont > 0 && workers > 2 )) {
+    if( miners > 0  || ( cont > 0 && workers > room.Setup['worker'].maxCount(room) ) ) {
```

I do not know what I mean.  I kept being told to change that line but had no idea what to change it to.  If that works and returns the currently calculated maximum value for Workers in the room then please apply it. I can type all kind of combinations into that code line that silently fail, or pass, or who knows what so there is little value in my writing anything off the top of my head.